.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "NHS Prototype Kit",
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  // codespaces seems to have an issue using port 2000 so setting to 2001 for the NHS Prototype Kit
+  "runArgs": ["--env", "PORT=2001"],
```

Comment:
One to look into for later. Either we could fix the issue, or possibly change the default port?

Reply:
Ah.. Good spot! I thought that was working but I might have made that up because it does the same for me.